doc/api/cli.md

```diff
@@ -898,7 +898,8 @@ changes:
 > [`--import` with `register()`][module customization hooks: enabling] instead.
 
 Specify the `module` containing exported [module customization hooks][].
-`module` may be any string accepted as an [`import` specifier][].
+`module` may be any string accepted as an [`import` specifier][]. This option
+has no effect on worker threads.
```

Comment:
To me this reads as “the loader won’t apply to worker threads,” but what I assume you mean is “you don’t need to pass this argument when spawning a worker thread, as it will inherit from the main thread.” I’m not sure we need this addition at all; the other addition in `module.md` might be enough.

Reply:
indeed that was intended. What do you think about: "This option
has no effect on worker threads. Worker threads inherit the customization hooks chain configured on main thread."

I can also remove this addition. The idea for adding it here as well was to have all relevant entry points in the documentation covered. To make it clear so that the user expectation is correct.

Reply:
I updated the commit with the last proposal. Let me know what you think

Reply:
@GeoffreyBooth friendly ping, trying to move this one forward. Is the current version suitable for landing?